tests/test_rio_merge.py

```diff
@@ -19,7 +23,7 @@
 def test_data_dir_1(tmpdir):
     kwargs = {
         "crs": {'init': 'epsg:4326'},
-        "transform": (-114, 0.2, 0, 46, 0, -0.2),
+        "transform": affine.Affine.from_gdal(-114, 0.2, 0, 46, 0, -0.2),
```

Comment:
@geowurster open question - should the parameters be reordered so that you can just create them as `Affine` objects without importing from GDAL-style transforms?  I'm thinking that it would be good for our tests to model our preferred approach.